code/modules/mining/lavaland/seismicarm.dm

```diff
@@ -315,22 +295,23 @@
 	icon = 'icons/mob/augmentation/augments_seismic.dmi'
 	icon_state = "seismic_r_arm"
 	max_damage = 60
-	var/list/seismic_arm_moves = list(
-		/datum/action/cooldown/spell/pointed/seismic/lariat,
-		/datum/action/cooldown/spell/pointed/seismic/mop,
-		/datum/action/cooldown/spell/pointed/seismic/suplex,
-		/datum/action/cooldown/spell/touch/righthook,
-	)
+	var/datum/action/cooldown/seismic/lariat/lariat = new/datum/action/cooldown/seismic/lariat()
+	var/datum/action/cooldown/seismic/mop/mop = new/datum/action/cooldown/seismic/mop()
+	var/datum/action/cooldown/seismic/suplex/suplex = new/datum/action/cooldown/seismic/suplex()
+	var/datum/action/cooldown/seismic/righthook/righthook = new/datum/action/cooldown/seismic/righthook()
```

Comment:
why? this just makes adding new abilities worse

Reply:
i don't like them being spells

Reply:
it doesn't require them being spells for the list to work